apps/fluent-tester/src/FluentTester/TestComponents/Notification/NotificationTest.tsx

```diff
@@ -0,0 +1,183 @@
+import * as React from 'react';
+// import { Notification } from
+import { View, Text } from 'react-native';
+import { Test, TestSection, PlatformStatus } from '../Test';
+// import { ColorValue } from 'react-native';
+
+
+const PrimaryToastTest: React.FunctionComponent = () => {
+
+  const styles = {
+    container: {
+      width: null,
+      backgroundColor: 'skyblue',
+      borderRadius: 12,
+
+      flexDirection: 'row',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+
+      padding: 16,
+    },
+    left: {
+      fontSize: 16,
+      flexDirection: 'row',
+      justifyContent: 'flex-start',
+    },
+    right: {
+      fontSize: 16,
+      fontWeight: '500',
+      flexDirection: 'row',
+      justifyContent: 'flex-end',
+    },
```

Comment:
A better name for these would be "start" and "end". This is because the order of the content can flip in a Right-To-Left (RTL) language, like Hebrew or Arabic. Notice that `justify-content` on lines 25 and 31 use `flex-start` and flex-end` for this reason.